prisma/schema.sql

```diff
@@ -71,10 +71,10 @@ CREATE TABLE physicians (
   address_line_1  VARCHAR(255) NOT NULL,
   address_line_2  VARCHAR(255),
   city            VARCHAR(255) NOT NULL,
-  province        Province NOT NULL,
+  province        Province NOT NULL DEFAULT 'BC',
```

Comment:
Upserting a physician was failing because province and status are not specified in `EditDoctorInformationModal`. The default values could have been specified somewhere else instead but I think `BC` and `ACTIVE` are sensible defaults to have on the database level so we don't have to specify these values